.buildkite/scripts/pipelines/security_solution_quality_gate/create_docker_image.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+if [ -z "${KIBANA_CURRENT+x}" ] || [ "$KIBANA_CURRENT" = "0" ]; then
+    echo "As we not testing against latest kibana image, this step is exiting with exit code 0"
+    exit 0
+fi
+
+
+.buildkite/scripts/bootstrap.sh
+
+source .buildkite/scripts/steps/artifacts/env.sh
+
+GIT_ABBREV_COMMIT=${BUILDKITE_COMMIT:0:12}
+KIBANA_IMAGE_TAG="git-$GIT_ABBREV_COMMIT"
+
+
+KIBANA_BASE_IMAGE="docker.elastic.co/kibana-ci/kibana-serverless"
+export KIBANA_IMAGE="$KIBANA_BASE_IMAGE:$KIBANA_IMAGE_TAG"
+
+echo "--- Verify manifest does not already exist"
+echo "$KIBANA_DOCKER_PASSWORD" | docker login -u "$KIBANA_DOCKER_USERNAME" --password-stdin docker.elastic.co
+trap 'docker logout docker.elastic.co' EXIT
+
+echo "Checking manifest for $KIBANA_IMAGE"
```

Comment:
Can we reuse the already built serverless images?

If not and we go with the rebuild route, we're going to want to change the image tag or remove `exit 0`.  This check will cause an early exit here or in the standard build and upload step, causing one of them to fail.

Reply:
We decided to duplicate and have a different image name.